*** ./smali/android/preference/RingtonePreference.smali	2013-03-13 08:05:13.870153000 +0800
--- /home/scue/patchrom_ics/stuttgart/temp/new_smali/framework.jar.out/./smali/android/preference/RingtonePreference.smali	2013-03-13 08:01:47.594739000 +0800
***************
*** 311,316 ****
--- 311,322 ----
  
      invoke-virtual {p1, v0, v1}, Landroid/content/Intent;->putExtra(Ljava/lang/String;Ljava/lang/CharSequence;)Landroid/content/Intent;
  
+     const-string v0, "com.android.thememanager"
+ 
+     const-string v1, "com.android.thememanager.ThemeResourceTabActivity"
+ 
+     invoke-virtual {p1, v0, v1}, Landroid/content/Intent;->setClassName(Ljava/lang/String;Ljava/lang/String;)Landroid/content/Intent;
+ 
      return-void
  .end method
  
