*** ./smali/com/android/internal/view/menu/ActionMenuItemView.smali	2013-03-13 08:06:51.435961000 +0800
--- /home/scue/patchrom_ics/stuttgart/temp/new_smali/framework.jar.out/./smali/com/android/internal/view/menu/ActionMenuItemView.smali	2013-03-13 08:03:08.349704000 +0800
***************
*** 133,138 ****
--- 133,146 ----
  
      iget-object v3, p0, Lcom/android/internal/view/menu/ActionMenuItemView;->mItemData:Lcom/android/internal/view/menu/MenuItemImpl;
  
+     invoke-virtual {v3}, Lcom/android/internal/view/menu/MenuItemImpl;->isForceShowText()Z
+ 
+     move-result v3
+ 
+     if-nez v3, :cond_0
+ 
+     iget-object v3, p0, Lcom/android/internal/view/menu/ActionMenuItemView;->mItemData:Lcom/android/internal/view/menu/MenuItemImpl;
+ 
      invoke-virtual {v3}, Lcom/android/internal/view/menu/MenuItemImpl;->showsTextAsAction()Z
  
      move-result v3
